README.rst

```diff
@@ -57,13 +57,31 @@ Django’s settings.py require the following format to connect to an Informix da
         'PASSWORD': 'passw0rd',
         'OPTIONS': {
             'DRIVER': '/path/to/iclit09b.so'. # Or iclit09b.dylib on macOS
+            'CPTIMEOUT': 120,
```

Comment:
I know it's not related to this patch, but could you please add some words explaining what a "CPTIMEOUT" is?

Reply:
Added.